src/main/kotlin/com/epam/brn/model/StudyHistory.kt

```diff
@@ -25,4 +25,8 @@ data class StudyHistory(
     val doneTasksCount: Short,
     val successTasksCount: Short,
     val repetitionCount: Short
-)
\ No newline at end of file
+) {
+    override fun toString(): String {
```

Comment:
it's not about your code, but about StudyHistory entity, what do you think about the separate sequence for id generation, from my point of view thats good option using postgres+ hibernate
    @Id
    @GeneratedValue(generator = "study_history_id_seq", strategy = GenerationType.SEQUENCE)
    @SequenceGenerator(
        name = "study_history_id_seq",
        sequenceName = "study_history_id_seq",
        allocationSize = 50
    ) 
something like this. What do you think?

Reply:
added